### PR TITLE
*: add pd-recover to Dockerfile

### LINKED ISSUE #2402: pd-recover should be included in the pd docker image
### Describe your feature request related problem: 

There is no dedicated download link for pd-recover and no binary in the docker image, it's not easy for Kubernetes users to get the pd-recover binary.
### Describe the feature you'd like:

pd-recover should be included in the pd docker image.

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve? 
Closes #2402.

### What is changed and how it works?
This PR does three things:
- change `make` target to compile pd-recover together
- add pd-recover to the docker image
- add `-V` parameter to show the version of pd-recover

### Check List 

Tests 

 - Unit test

### Release note 
- Fix the issue that pd-recover is not included in the docker image

### CODE REVIEW

**Comment by shenli** [comment]
@rleungx Please cherry-pick this to 3.1. Thanks!

**Comment by rleungx** [comment]
> @rleungx Please cherry-pick this to 3.1. Thanks!

OK

File: tools/pd-recover/main.go [hunk 1]
```diff
@@ -39,26 +41,45 @@ func exitErr(err error) {
...
 
 func main() {
-	flag.Parse()
-	if *clusterID == 0 {
+	fs := flag.NewFlagSet("pd-recover", flag.ExitOnError)
+	fs.BoolVar(&v, "V", false, "print version information.")
+	fs.StringVar(&endpoints, "endpoints", "http://127.0.0.1:2379", "endpoints urls")
+	fs.Uint64Var(&allocID, "alloc-id ", 0, "please make sure alloced ID is safe")
+	fs.Uint64Var(&clusterID, "cluster-id", 0, "please make cluster ID match with tikv")
+	fs.StringVar(&caPath, "cacert", "", "path of file that contains list of trusted SSL CAs.")
```
Comment (HunDunDM) [inline tools/pd-recover/main.go:49]:
```suggestion
	fs.StringVar(&caPath, "cacert", "", "path of file that contains list of trusted SSL CAs")
```

File: tools/pd-recover/main.go [hunk 2]
```diff
@@ -39,26 +41,45 @@ func exitErr(err error) {
...
+	fs.StringVar(&certPath, "cert", "", "path of file that contains list of trusted SSL CAs.")
```
Comment (HunDunDM) [inline tools/pd-recover/main.go:50]:
```suggestion
	fs.StringVar(&certPath, "cert", "", "path of file that contains list of trusted SSL CAs")
```

File: tools/pd-recover/main.go [hunk 3]
```diff
@@ -39,26 +41,45 @@ func exitErr(err error) {
...
+	fs.StringVar(&keyPath, "key", "", "path of file that contains X509 key in PEM format.")
```
Comment (HunDunDM) [inline tools/pd-recover/main.go:51]:
```suggestion
	fs.StringVar(&keyPath, "key", "", "path of file that contains X509 key in PEM format")
```

**Pull request merged**